.nvmrc

```diff
@@ -0,0 +1 @@
+v18.12.1
```

Comment:
should be v16 right ?

Reply:
v16.16.0 (LTS)

Reply:
Nice catch! Yes, it should! I'll change it right away